### PR TITLE
Add prettier as the code formatter

### PR DESCRIPTION
**Pull request by taylrj**

### CODE REVIEW

**Comment by taylrj** [comment]
unresoleved comment: https://github.com/twreporter/keystone/pull/180#discussion_r652598083

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (babygoat) [inline .jshintrc:1]: Why would we like to use two different types of linter?

**Comment by babygoat** [comment]
Do we want into include this PR on up-coming release?

**Comment by taylrj** [comment]
No, this change is not ready to be included.

File: .jshintrc (hunk 1 above)
Comment (taylrj) [reply .jshintrc:1]: `jshint` is utilized in huge amounts legacy codes.

The .jshintignore, .jshintrc files are auto generated.

I'll remove them in this change.

**Comment by taylrj** [comment]
related comment: [#180 (comment)](https://github.com/twreporter/keystone/pull/180#discussion_r652598083)

> I don't have a strong opinion here. However, to avoid inconsistent style, we need to either use an automatic style fixer (e.g liner --fix) or have clear style instructions. One more question, so the prettier should be used along with eslint or jshint?

It seems `eslint --fix` can fix the mix spaces and tabs properly in v7.3.0 (see [this diff](https://github.com/twreporter/keystone/pull/193)). So I guess we don't need Prettier since it is introduced here for resolving this issue.

**Comment by taylrj** [comment]
Closing this PR since we are not going to adopt Prettier as the style fixer.

**Pull request closed without merging**